scripts/metadata_table.py

```diff
@@ -223,6 +223,51 @@ def parse_metadata_file(filename, known_ddts, run_env, skip_ddt_check=False):
 
 ########################################################################
 
+def find_module_name(filename):
```

Comment:
I am surprised that this isn't already done by the parser. Is that because everything is based on the assumption filename=modulename?

Reply:
Yeah. Metadata tables for new DDTs default to the source filename (e.g. `[tests](https://github.com/NCAR/ccpp-framework/blob/develop/test/capgen_test/test_host_data.meta)`). Not the best.
This is a problem for DDTs in two instances, a) DDTs housed in differing module/file names and b) DDTs in files with multiple modules.
We don't have any of the latter in the SCM/UFS, so I haven't gone down that rabbit hole.